Doc/library/xml.etree.elementtree.rst

```diff
@@ -665,6 +665,11 @@ Functions
    not given, the standard :class:`XMLParser` parser is used.  Returns an
    :class:`ElementTree` instance.
 
+   .. warning::
+
+      When the *source* encoding is ``ISO-8859-1`` and the mode is ``r``, An encoding
+      error warning will be thrown.
```

Comment:
```suggestion
   .. note::

      When the *source* encoding is ``ISO-8859-1`` and the mode is ``r``,
      a warning is emited.
```

Reply:
Oh, I was wondering if using warnings would be better. But now I know.

Reply:
I don't think that ``.. warnings::`` is justified here, it's not a major issue.